### PR TITLE
Branch update help

### PR DESCRIPTION
**Pull request by a1021492980**
This PR contains the updated version of help feature

### CODE REVIEW

File: src/main/java/werkit/UI.java [hunk 1]
```diff
@@ -224,15 +224,20 @@ public String getColorText(String color, String text) {
...
         printLine();
-        printWorkoutUpdateHelp();
+        printPlanHelp();
+        printLine();
+        printLine();
+        printScheduleHelp();
+        printLine();
+        printLine();
+        printSearchHelp();
+        printLine();
```
Comment (Musfirahe0556596) [inline src/main/java/werkit/UI.java:240]: Nice job separating and generalizing the methods!

File: src/main/java/werkit/UI.java [hunk 2]
```diff
@@ -495,4 +504,200 @@ public void printClearedScheduleMessage() {
...
+    }
+
+    /**
+     * Prints help message for 'plan /list' command.
+     */
+    public void printPlanListHelp() {
+        System.out.println("\t To view all plans, please enter:");
+        printColorText(TextColor.COLOR_YELLOW, "\t plan /list");
+        System.out.println("\t This will print all the existing plans.");
+    }
```
Comment (Musfirahe0556596) [inline src/main/java/werkit/UI.java:562]: don't forget to add in the plan /details to view a plan in more details once implemented !

**Review by Musfirahe0556596 — APPROVED** [review]
Looks good! thank you!

File: src/main/java/werkit/UI.java (hunk 1 above)
Comment (a1021492980) [reply src/main/java/werkit/UI.java:240]: Thank you!

File: src/main/java/werkit/UI.java (hunk 2 above)
Comment (a1021492980) [reply src/main/java/werkit/UI.java:562]: I have added this into the help message, thank you!

**Pull request merged**